albumentations/augmentations/crops/transforms.py

```diff
@@ -502,13 +502,13 @@ class RandomCropNearBBox(DualTransform):
     def __init__(
         self,
         max_part_shift: ScaleFloatType = (0.3, 0.3),
-        cropping_box_key: str = "cropping_bbox",
+        cropping_bbox_key: str = "cropping_bbox",
```

Comment:
This will might break backward compatibility. Are you sure that this is a good idea - rename this parameter?

Reply:
I do not like that naming is confusing across the class, but you are right.

Let's do it smoother, through the deprecation warning.